src/library/scala/DelayedInit.scala

```diff
@@ -43,7 +43,7 @@ package scala
  *
  *  @author  Martin Odersky
  */
-@deprecated("DelayedInit semantics can be surprising. Support for `App` will continue. See the release notes for more details: https://github.com/scala/scala/releases/tag/v2.11.0-RC1", "2.11.0")
+@deprecated("DelayedInit semantics can be surprising. Support for `App` will continue. See the release notes for more details: [[https://github.com/scala/scala/releases/tag/v2.11.0]]", "2.11.0")
```

Comment:
I'm sorry to see this only now, but could you remove the `[[` / `]]` link markers here? It's not in a Scaladoc comment and will show up on the command line (deprecation warning). Then we also don't need the change to  test/files/neg/delayed-init-ref.check.

Reply:
No worries!  I'm pretty the links **do** show up [in the Scaladoc](http://www.scala-lang.org/api/2.12.3/scala/DelayedInit.html), but sure, I agree the brackets shouldn't show up on the console output!  I'll undo both the change and the check.